### PR TITLE
HICU: hotfixes

### PR DESCRIPTION
**Pull request by Doster-d**
## About The Pull Request

Fix bugs found out in Friday night.

## Why It's Good For The Game

Bug fixes are good.

## Changelog
:cl:
fix: Implant circuits are now work fine.
fix: Chemical syphon are now have sprites.
fix: Used correct material cost multiplication.
add: Cloning design are now avaliable in research.
tweak: Added new tools to research design (they are in printer too).
tweak: Now it's possible to clone implant circuit.
/:cl:

### CODE REVIEW

**Comment by Doster-d** [comment]
If anyone know how to fix image in reasearch datum, please fix it.

File: code/__DEFINES/research.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#define SHEET_MATERIAL_AMOUNT 1
+#define SHEET_MATERIAL_AMOUNT 2000
```
Comment (TheShown911) [inline code/__DEFINES/research.dm:1]: Does this only affect the circuits, or does it affect other research?

**Review by TheShown911 — APPROVED** [review]

File: code/__DEFINES/research.dm (hunk 1 above)
Comment (Doster-d) [reply code/__DEFINES/research.dm:1]: It's affect only circuit, it was commented before me.

**Comment by Doster-d** [comment]
Fork I want to post my fix for examiner.
Well I need to create a new pr.

**Pull request merged**